app/src/main/assets/pages/Premium.html

```diff
@@ -340,22 +357,25 @@
                     $('#spBalance').text(Balance);
                     $('#spContribution').text(Contribution);
                 }else{
-                    photoValue = 1;
-                    var Balance = parseInt($('#hfBalance').val());
                     var Contribution = parseInt($('#hfContribution').val());
-                    $('#spBalance').text(Balance);
+                
+                    $('#spBalance').text(pp - parseInt($('#txtAmount').val()));
                     if(Balance < 0){
```

Comment:
Could you just confirm that it is normal to use Balance to assign bb ?

Reply:
I'm not sure about this one. Can you explain more precisely what you mean by using Balance to assign bb?

Reply:
I find this code strange. It seems the Balance variable is created when `premiumId != 0`, means modifying a contribution. If new contribution, then there should be a javascript error.